README.md

```diff
@@ -1 +1,78 @@
-# codacy-test
\ No newline at end of file
+# Codacy ESLint
+
+## Adding new plugins / configs
+
+1.  Install the package using npm
+2.  \[Plugins only\] Add the plugin to the plugins section in the file `src/eslintDefaultOptions.ts`
+3.  \[Plugins only\] If the plugin has descriptions for rules on GitHub, reference them
+at `src/docGeneratorMain.ts` to include them on the generated documentation. To do this, add a section similar to the following example:
+
+    ```typescript
+    console.log("Generate xss description files")
+    await docGenerator.downloadDocs(
+      (pattern) =>
+        `${githubBaseUrl}/Rantanen/eslint-plugin-xss/master/docs/rules/${pattern}.md`,
+      "xss"
+    )
+    ```
+
+4.  Generate documentation so it adds the new plugin documentation
+
+## Generating documentation
+
+```bash
+npm install
+npm run generateDocs
+```
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion

## Test changes to codacy-seed locally
```